helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterModelProvider.java

```diff
@@ -391,6 +406,34 @@ private static Set<AssignableReplica> findToBeAssignedReplicasByClusterChanges(
     return toBeAssignedReplicas;
   }
 
+  private static Set<AssignableReplica> findToBeAssignedReplicasIllegalPlacements(
```

Comment:
nit: useful to have some high level comments for method.
if i understand, it is trying to find 'currentIllegalPlacements'